calico_node/startup/startup.go

```diff
@@ -84,14 +84,11 @@ func main() {
 	node := getNode(client, nodeName)
 
 	// Configure and verify the node IP addresses and subnets.
-	configureIPsAndSubnets(node)
+	configureIPsAndSubnets(client, node)
```

Comment:
minor nit - rather than pass the client here it might be nicer to make it so `configureIPsAndSubnets` returns whether or not the an IP on the node changed, and we can use it below.